pages/login.vue

```diff
@@ -1,20 +1,30 @@
 <template>
   <section v-if='!requestedLogin' class="min-h-screen flex items-center justify-center">
-    <div>
-      <h1 class="text-4xl font-bold mb-16">Start tracking your mood</h1>
-      <div class="mb-4">
-        <label class="block" for="">Mail</label>
-        <input v-model="mail" class="block bg-gray-600 text-white rounded shadow-brutal-sm w-full p-2" type="text" />
-      </div>
-      <div>
-        <button class="bg-green-500 block font-bold rounded shadow-brutal-sm w-full py-2" @click="loginUser()">
-          Login
-        </button>
+    <div class='container max-w-6xl'>
+      <div class='md:grid md:grid-cols-2 md:gap-24'>
+        <div class='mb-16 md:mb-0'>
+          <h1 class="text-5xl font-extrabold mb-12">Start tracking your mood</h1>
+          <p class='text-xl'>emoodji helps you to spot when and why mood swings in your team occur. </p>
+        </div>
+        <div class='md:flex md:flex-col md:justify-center'>
+          <div class="mb-4">
+            <label class="block" for="">Mail</label>
+            <input v-model="mail" class="block bg-gray text-white rounded shadow-brutal-sm w-full p-2" type="text" />
+          </div>
+          <div>
+            <button class="bg-green block font-bold rounded shadow-brutal-sm w-full py-2" @click="loginUser()">
+              Login
+            </button>
+          </div>
+        </div>
       </div>
     </div>
   </section>
   <section v-else class="min-h-screen flex items-center justify-center">
-    <p class='text-center'>We sent you a magic link to log in to emoodji. Just check your mails.</p>
+    <div class='text-center'>
+      <h2 class='text-3xl font-extrabold mb-3'>It's magic!</h2>
+      <p class='text-center'>We sent you a magic link to log in to emoodji. Just check your mails.</p>
+    </div>
   </section>
 </template>
 
```

Comment:
import { ref } from 'vue' can be deleted as it will be auto-imported.

Reply:
Done!